test/gtest/uct/uct_test.cc

```diff
@@ -417,7 +417,9 @@ std::vector<const resource*> uct_test::enum_resources(const std::string& tl_name
 
             uct_md_attr_t md_attr;
             status = uct_md_query(md, &md_attr);
-            ASSERT_UCS_OK(status);
+            if (status != UCS_OK) {
```

Comment:
should cause posix to not appear in enum_md_resources output when it's disqualified